Formula/samba.rb

```diff
@@ -7,6 +7,7 @@ class Samba < Formula
   url "https://download.samba.org/pub/samba/stable/samba-4.14.5.tar.gz"
   sha256 "bb6ef5d2f16b85288d823578abc453d9a80514c42e5a2ea2c4e3c60dc42335c3"
   license "GPL-3.0-or-later"
+  revision 2
```

Comment:
```suggestion
  revision 1
```
`revision` starts at `0`.

Reply:
fixed